src/utils/signInWithEthereum.ts

```diff
@@ -75,10 +75,7 @@ const signInWithEthereum = async ({
     );
 
     // Update global account data
-    updateAccountInfo({
-      account: account,
-      isVerified: isValidSignature === EIP_1271_MAGIC_VALUE,
-    });
+    updateVerification(isValidSignature === EIP_1271_MAGIC_VALUE);
```

Comment:
this feels like comparing a boolean to a string (the isValidSignature name suggests it holds a bool, but it holds a string)

Not a big deal but im sharing anyway